tests/nested/core/core20-kernel-failover/task.yaml

```diff
@@ -40,9 +64,20 @@ execute: |
 
   echo "Install it and get the ID for the change"
   REMOTE_CHG_ID=$(tests.nested exec sudo snap install --dangerous panicking-initramfs-kernel.snap --no-wait)
+  if [ "$PROBLEM_TYPE" = "nolink" ]; then
+      while ! tests.nested exec test -f /run/mnt/ubuntu-boot/EFI/ubuntu/try-kernel.efi
+      do sleep 5
+      done
+      tests.nested exec sudo rm /run/mnt/ubuntu-boot/EFI/ubuntu/try-kernel.efi
+  fi
 
-  # wait for a reboot
-  tests.nested wait-for reboot "${boot_id}"
+  # wait for a reboot. Note that failure is immediate for kernel.efi
+  # files with invalid PE headers in the FDE case, as snapd is not
+  # able to reseal the encryption key in that case ("cannot compute
```

Comment:
is reseal here meant to be unseal?

Reply:
Right, it is "unseal" indeed, changing that now.